packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -65,14 +67,14 @@ const Encryption = WebexPlugin.extend({
         .then((result) => result.plaintext.toString()));
   },
 
-  download(scr) {
+  download(scr, optional) {
     /* istanbul ignore if */
     if (!scr.loc) {
       return Promise.reject(new Error('`scr.loc` is required'));
     }
 
     const shunt = new EventEmitter();
-    const promise = this._fetchDownloadUrl(scr)
+    const promise = this._fetchDownloadUrl(scr, optional)
```

Comment:
The general naming convention for this param would be `options`. `optional` would be more of a boolean value and this looks like it takes an object. Also, adding some code comments or js-docs would greatly explain why/what is happening